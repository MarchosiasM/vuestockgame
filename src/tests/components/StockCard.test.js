import { mount } from '@vue/test-utils';
import StockCard from '../../Components/StockCard.vue';

describe('Stock card', () => {
  const wrapper = mount(StockCard);
  it('renders properly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
