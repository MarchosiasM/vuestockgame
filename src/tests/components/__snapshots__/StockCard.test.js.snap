// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stock card renders properly 1`] = `<div />`;
